### PR TITLE
Added MacPorts Configuration

### PR DESCRIPTION
**Pull request by ZroMnc**

### CODE REVIEW

**Comment by hjacobs** [comment]
@cgmv thanks for the PR --- could you update your fork please (it's out of date => merge conflcts)

**Comment by hjacobs** [comment]
@cgmv looks there is still an unresolved conflict:
+>>>>>>> d554e968f6cf311ff2464bc8a36756105b8a4e86

File: user-guide/local-setup.rst [hunk 1]
```diff
@@ -42,12 +56,72 @@ Install Python 3 using Homebrew (pip3 already comes with this package)
...
+Docker
+------
+=======
 OS X users may need to set their locale environment to UTF-8 with::
 
     export LC_ALL=en_US.utf-8
     export LANG=en_US.utf-8
 
 You can put these two commands in your local shell initialization script, e.g. ``.bashrc``.
+>>>>>>> d554e968f6cf311ff2464bc8a36756105b8a4e86
```
Comment (tuxlife) [inline user-guide/local-setup.rst]: @cgmv please remove this line

**Comment by ZroMnc** [comment]
@hjacobs should be fixed now

**Pull request merged**